lib/rules/cli/4.1.yaml

```diff
@@ -391,6 +391,10 @@
     :template: --template=<value>
     :type: --type=<value>
     :wildcardpolicy: --wildcard-policy=<value>
+:exec_extension_oc_cmd:
+  :cmd: oc
+  :options:
+    :arg: <value>
```

Comment:
Can we use the lib which is already existing which looks similar to this ? Below is what i see can be used instead of creating this new one. what do you think  ?

exec_raw_oc_cmd_for_neg_tests

Reply:
Then we should use `When I run the :exec_raw_oc_cmd_for_neg_tests client command with:`   
that's make no sense.

Reply:
May i know why it does not make any sense ? I see it is very similar to the one you have written here but just the name is not suitable.

:exec_raw_oc_cmd_for_neg_tests:   # used for negative testing in which we want to deviate from yaml
  :cmd: oc
  :options:
    :arg: <value>
    :test_do_not_use: <value>

Reply:
@kasturinarra Have updated, make it more readable.

Reply:
Perhaps a scenario with the command can help us understand the command.